src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/grammar/antlr.bat

```diff
@@ -0,0 +1 @@
+java org.antlr.v4.Tool %*
```

Comment:
I do have a question though, would it be desired to make the scripts platform-independent?

Reply:
Philosophically, yes. Practically, no. Regardless of your dev environment, if you need to modify the grammar, you have to do a bunch more manual setup to be able to generate the grammar code anyways.